models/quality_measures/intermediate/cbe0101_falls_plan_of_care/quality_measures__int_cbe0101_numerator.sql

```diff
@@ -0,0 +1,147 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False))))
+    | as_bool
+   )
+}}
+
+with denominator as (
+
+    select 
+          patient_id
+        , encounter_date
+        , performance_period_begin
+        , performance_period_end
+        , measure_id
+        , measure_name
+        , measure_version
+    from {{ ref('quality_measures__int_cbe0101_denominator') }}
+
+)
+
+, fallcare_codes as (
+
+    select
+          code
+        , code_system
+        , concept_name
+    from {{ ref('quality_measures__value_sets') }}
+    where code = '0518F' -- Falls plan of care documented
+
+)
+
+, procedures as (
+
+    select
+        patient_id
+      , procedure_date
+      , coalesce (
+              normalized_code_type
+            , case
+                when lower(source_code_type) = 'cpt' then 'hcpcs'
+                when lower(source_code_type) = 'snomed' then 'snomed-ct'
+                else lower(source_code_type)
+              end
+          ) as code_type
+        , coalesce (
+              normalized_code
+            , source_code
+          ) as code
+    from {{ ref('quality_measures__stg_core__procedure') }}
+
+)
+
+, qualifying_procedures as (
+
+    select
+          patient_id
+        , procedure_date as evidence_date
+    from procedures
+    inner join fallcare_codes
+        on procedures.code = fallcare_codes.code
+            and procedures.code_type = fallcare_codes.code_system
+            
+)
+
+, qualifying_claims as (
+
+    select
+          patient_id
+        , coalesce(claim_end_date, claim_start_date) as evidence_date
+    from {{ ref('quality_measures__stg_medical_claim') }} medical_claim
+    inner join fallcare_codes
+        on medical_claim.hcpcs_code = fallcare_codes.code
+            and lower(fallcare_codes.code_system) = 'hcpcs'
+
+)
+
+, qualifying_cares as (
+
+    select
+          patient_id
+        , evidence_date
+    from qualifying_procedures
+
+    union all
+
+    select
+          patient_id
+        , evidence_date
+    from qualifying_claims
+
+)
+
+, combined_qualifying_patients as (
+
+    select
+          qualifying_cares.patient_id
+        , qualifying_cares.evidence_date
+        , null as evidence_value
```

Comment:
This null placeholder needs to be casted to prevent errors in Redshift.

```suggestion
        , cast(null as {{ dbt.type_string() }}) as evidence_value
```